content/en/docs/kuberay_on_volcano.md

```diff
@@ -0,0 +1,124 @@
+
+#### Supported Labels
+
+To configure RayJob and RayCluster resources with Volcano scheduling, you can use the following labels in the metadata section:
+
+| Label | Description | Required |
+|-------|-------------|----------|
+| `ray.io/priority-class-name` | Assigns a [Kubernetes](https://kubernetes.io/docs/concepts/scheduling-eviction/pod-priority-preemption/#priorityclass) priority class for pod scheduling | No |
+| `volcano.sh/queue-name` | Specifies the Volcano queue for resource submission | No |
+| `volcano.sh/network-topology-mode` | Configures network topology-aware scheduling mode | No |
+| `volcano.sh/network-topology-highest-tier-allowed` | Sets the highest network tier allowed for scheduling | No |
+
+Below are setup examples with detailed explanations. For comprehensive configuration options, please refer to the [KubeRay Volcano Scheduler Documentation](https://docs.ray.io/en/latest/cluster/kubernetes/k8s-ecosystem/volcano.html#kuberay-integration-with-volcano).
+
+#### Autoscaling Behavior
+
+KubeRay's integration with Volcano handles gang scheduling differently based on whether autoscaling is enabled:
+
+- **When autoscaling is enabled**: `minReplicas` is used for gang scheduling
+- **When autoscaling is disabled**: The desired replica count is used for gang scheduling
+
+This ensures that the gang scheduling constraints are properly maintained while allowing for flexible scaling behaviors based on your workload requirements.
+
+
+### Setup Guide
+
+#### Prerequisites
+
+##### 1. Create a Kubernetes Cluster
+```bash
+$ kind create cluster
+```
+
+##### 2. Install Volcano
+Follow the instructions in the [Volcano installation guide](https://volcano.sh/en/docs/installation/) to install Volcano on your Kubernetes cluster.
+
+##### 3. Install KubeRay Operator
+Deploy the KubeRay Operator with the `--batch-scheduler=volcano` flag to enable Volcano batch scheduling support:
+```bash
+$ helm install kuberay-operator kuberay/kuberay-operator --version 1.5.1 --set batchScheduler.name=volcano
+```
+
+#### Example Deployments
+
+##### RayCluster Example
+
+Deploy a RayCluster with Volcano scheduling:
+
+```bash
+# Download the sample RayCluster configuration with Volcano labels
+$ curl -LO https://raw.githubusercontent.com/ray-project/kuberay/v1.5.1/ray-operator/config/samples/ray-cluster.volcano-scheduler.yaml
+
+# Apply the configuration
+$ kubectl apply -f ray-cluster.volcano-scheduler.yaml
+
+# Verify the RayCluster deployment
+$ kubectl get pod -l ray.io/cluster=test-cluster-0
+
+# Expected output:
+# NAME                                 READY   STATUS    RESTARTS   AGE
+# test-cluster-0-head-jj9bg            1/1     Running   0          36s
+```
+
+##### RayJob Example
```

Comment:
Could you also add vcjob example? https://github.com/volcano-sh/volcano/blob/master/docs/user-guide/how_to_use_ray_plugin.md, in volcano v1.13 version, we have also supported using vcjob to create raycluster